### ISSUE TITLE
push on /notice command

### ISSUE THREAD

**Issue by strife84** [jreese/znc-push#242]
Hi, thank you for this great module. It works very well on telegram. I have just a question (because I'm a total noob at programming) and I don't know if this is the right section. If not, sorry. Does znc push work with /notice command? For example, if someone executes "/notice mynickname message" on irc, does znc push send me a notification? thank you

**Comment by strife84** [comment]
anyone?

**Comment by jreese** [comment]
I don't believe it does, but I'm not 100% sure how ZNC exposes notices to plugins, as I've never used them in this context.

**Comment by strife84** [comment]
that would be great!

**Comment by dgw** [comment]
@strife84 Want to give #243 a try and let me/us know if it does what you're looking for?

**Comment by strife84** [comment]
it works like a charm on private notices. thank you so much!
